packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
+    Given a user is on the Indirect Tax Form
+
+  @minutia
+  Scenario Outline: Different indirect tax inputs are shown depending on the country of a participant
```

Comment:
I find this spec a bit confusing with some of the others in this file. It might be simpler to have 3 specs
Spec one: what happens when someone selects Canada in step 1 and lands here
Spec two: What happens when someone selects a country that isn't Canada but has indirect tax handling (and list these countries)
Spec three: What happens when someone selects a country that doesn't have indirect tax handling (and list these countries)

Reply:
@Locrian24 @AndiLuo